crates/wasi-common/src/macros.rs

```diff
@@ -2,12 +2,12 @@ macro_rules! hostcalls {
     ($(pub unsafe fn $name:ident($($arg:ident: $ty:ty,)*) -> $ret:ty;)*) => ($(
             #[wasi_common_cbindgen::wasi_common_cbindgen]
             pub unsafe fn $name($($arg: $ty,)*) -> $ret {
-                let ret = match hostcalls_impl::$name($($arg,)*) {
-                    Ok(()) => wasi::__WASI_ERRNO_SUCCESS,
-                    Err(e) => e.as_wasi_errno(),
-                };
-
-                ret
+                let ret = crate::hostcalls_impl::$name($($arg,)*)
+                    .err()
+                    .unwrap_or(crate::Error::ESUCCESS)
+                    .as_wasi_error();
+                log::trace!("\t | errno={}", ret);
```

Comment:
The rest of the codebase is not using `\t` but `    `. This is indeed much more readable, but should be consistent across the codebase. But it's ok to deal with it in a subsequent PR.

Reply:
Great catch, thanks! I'll make sure to update it!

Reply:
Agreed; regardless of the merit of tabs, using spaces would allow this to match the indentation level used for similar messages in wasmtime's logging.